docs/topics/device_connection.md

```diff
@@ -0,0 +1,265 @@
+`/gardens` have been created. All the requests to the platform will include the appropriate ´Fiware-service´ and
+`Fiware-servicepath` headers, as explained [here](../multitenancy.md).
+
+## Sending a measure to the Platform
+
+### Configuring the APIKey and protocol
+
+In order for the devices to send information for the subservice, a Configuration Group must be created. This configuration
+group will specify which protocol the devices will be using, and the APIKey that will be used to identify that the devices
+belong to that particular group.
+
+The following request will create the required group for our subservice:
+
+    POST /iot/services
+    Content-Type: application/json
+    Fiware-service: smartown
+    Fiware-servicepath: /gardens
+
+    {
+      "services": [
+        {
+          "protocol": [
+                  "IoTA-UL"
+                ],
+          "apikey": "801230BJKL23Y9090DSFL123HJK09H324HV8732",
+          "entity_type": "Device"
+        }
+      ]
+    }
+
+This request specifies that the default type for the Context Entities representing our devices will be "Device", and
+that the protocol we will use for our devices will be Ultralight 2.0.
+
+### Provisioning the device
+
+The next step is provisioning the device in the platform. In this example, we will provision a single device: a Weather
+Station having two sensors, a humidity sensor and a thermometer. We can also use this provisioning to add some static
+information we know for our entity, that need not be reported by the device itself, but we want appearing in the
+entity itself. As en example, we will provision statically the Reference of the Weather Station and its location.
+
+To provision the device in the platfor we issue the following request to the IoTAgent:
+
+    POST /iot/devices
+    Content-Type: application/json
+    Fiware-service: smartown
+    Fiware-servicepath: /gardens
+
+    {
+      "devices": [
+        {
+          "device_id": "ws1956672",
+          "protocol": "IoTA-UL",
+          "entity_name": "WeatherStation1",
+          "entity_type": "Device",
+          "attributes": [
+            {
+              "object_id": "t",
+              "name": "temperature",
+              "type": "degrees"
+            },
+            {
+              "object_id": "h",
+              "name": "humidity",
+              "type": "percentage"
+            }
+          ],
+          "commands": [
+            {
+              "name": "configuration",
+              "type": "command"
+            }
+          ],
+          "static_attributes": [
+            {
+              "name": "location",
+              "type": "geo:point",
+              "value": "22.1, -17"
+            },
+            {
+              "name": "reference",
+              "type": "string",
+              "value": "917834508965243"
+            }
+          ]
+        }
+      ]
+    }
+
+This will provision our ws1 workstation in the system, making it ready to receive measures.
```

Comment:
Actually, I understand that it should be WeatherStation1, as that is the entity_name above.